lib/tasks/gettext.rake

```diff
@@ -30,13 +30,34 @@ namespace :gettext do
   end
 
   def find(files:, root:)
```

Comment:
Commit message doesn't say what was broken or why this fixes them, so it's pretty difficult to understand what to review.

Reply:
I'll expand on the commit message but basically as the same as https://github.com/mysociety/alaveteli/pull/5779/commits/ed4ca07a861be6c22fc0d00c7a9316c4727bb2d8 where upstream removed the helper methods we were using.